make_predictions.py

```diff
@@ -58,7 +60,10 @@
     'Path to model checkpoint. If a directory, the most '
     'recent model checkpoint in this directory will be used. If a file, it '
     'should be of the form /.../name-of-the-file.ckpt-10000')
-
+tf.flags.DEFINE_bool(
+    'make_and_save_spectra_plots', True,
```

Comment:
I would just call this 'save_spectra_plots'

Reply:
Done.